app/constants/orders/constants.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Orders
+  module Constants
+    module Payments
+      module Spree
```

Comment:
Don't we have these values in Spree already? Shouldn't we use them?

Reply:
I didn't find any constants in Spree with statuses, but I really want to use them instead of simple strings.
If you know where to find states constants please let me know

Reply:
I suppose you are right, spree keeps them as strings, as state_machine attributes. So if it is not you need, ok.

Reply:
Yes, Spree does not use this convention with constants for all strings values, especially when names are equals to values.  Sometimes they are used for configuration purposes, default values, etc. though. This looks just like Java thing.